source/_integrations/smtp.markdown

```diff
@@ -109,7 +109,7 @@ burglar:
 
 The optional `target` field is used to specify recipient(s) for this specific service call. When `target` field is not used, this message will be sent to default recipient(s), in this example, my_intruder_alert@example.com.
 
-The optional `images` field adds in-line image attachments to the email. This sends a text/HTML multi-part message instead of the plain text default.
+The optional `images` field adds image attachments to the email. If `html` is defined, the images will be added in-line. If `html` is not defined, images will be added as separate attachments.
```

Comment:
This seems to imply that if you add the html field then images and supporting html will be added inline automatically for you.

My understanding of the new behaviour makes me think that the following would be more accurate:

'If html is defined, this allows you to write your own html tags to define the location and properties of the images, in order to include them in the email'.

Reply:
Good point. I changed the order and attempted to further clarify. Please let me know if this reads better.